es6-lib/upstream/core.js

```diff
@@ -127,7 +125,7 @@ class Core extends CoreClient {
 
 
   getView(fourfour, onComplete, retried) {
-    const doGetView = () => this.getView(layer, onComplete, true);
+    const doGetView = () => this.getView(fourfour, onComplete, true);
```

Comment:
This seems like a big change, but I don't know what the implications are

Reply:
This should be a typo. In the scope of this function, `layer` does not exist, only `fourfour`. This line is also calling `this.getView(...)` which refers to the current function.